src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -53,6 +77,10 @@ const AdminDashboard = () => {
         openPremiumVideoCall(history)
     }
 
+    const getSMSUsage = () => details?.usage?.sentSMSs
```

Comment:
could be converted to selectors, 
getUserUsageSelector = state => getUserState(state).usage for example.